scss/standalone/patterns_grid-breakout.scss

```diff
@@ -0,0 +1,13 @@
+@import '../base';
```

Comment:
Is this file still needed or was it renamed to `layouts_fluid-breakout`?

Reply:
removed